css-color/Overview.bs

```diff
@@ -184,7 +184,7 @@ Resolving Color values</h2>
 		so descendants will use their own 'color' property to resolve it.
 
 		<li>
-		The computed value and used value of ''<color>/transparent'' is ''rgba(0,0,0,0)''.
+		The computed value and used value of ''<color>/transparent'' is ''rgba(0, 0, 0, 0)''.
```

Comment:
Maybe worth calling out explicitly that the serialization syntax (with commas) differs - for historical reasons - from the comma-less syntax which CSS4 color is now using (commas now being reserved for lists of alternatives).